shared/pages/Swap/Swap.js

```diff
@@ -87,7 +90,7 @@ export default class SwapComponent extends PureComponent {
           })
       })
 
-      window.swap = swap
+      this.swap = swap
```

Comment:
```suggestion
      window.swap = swap
```

это очень важная вещь для дебага, а this.swap это вообще что?